decentraland/components/presentational/LandForSaleList.test.js

```diff
@@ -6,11 +6,13 @@ describe("LandForSaleList", () => {
   it("renders the component", async () => {
     const landForSaleRenderer = () => {};
     const landForSaleList = [];
+    const loadingInProgress = true;
     expect(
       shallow(
         <LandForSaleList
           landForSaleList={landForSaleList}
           renderItem={landForSaleRenderer}
+          loadingInProgress={loadingInProgress}
```

Comment:
We probably want a separate test in this file that does a `loadingInProgress` `false` render

Reply:
Done.

Reply:
Thanks! Resolving